internal/core/src/bitset/detail/platform/x86/instruction_set.cpp

```diff
@@ -0,0 +1,139 @@
+// Licensed to the LF AI & Data foundation under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "instruction_set.h"
+
+#include <cpuid.h>
+
+#include <cstring>
+#include <iostream>
+
+namespace milvus {
+namespace bitset {
+namespace detail {
+namespace x86 {
```

Comment:
this file's content is more common, not only for bitset, in the future we may using simd for other interface, so put these content in bitset namespace is a good framework? maybe we should also keep simd parent directory and put biset in it? what do you think?

Reply:
I moved simd code there, because it was used for the bitset-related code only. Let's keep the file as is for now, and we'll move it away if there will be any other SIMD-based implementations of other components.